test/index.js

```diff
@@ -8,5 +8,12 @@ it('should find ffmpeg', function(t) {
 
   var stats = fs.statSync(ffmpegPath);
   t.ok(stats.isFile(ffmpegPath));
+
+  try {
+    fs.accessSync(ffmpegPath, fs.constants.X_OK)
+  } catch {
```

Comment:
Nice addition! 👍 Please use `t.doesNotThrow` to make this assertion show up in the TAP output even if it doesn't fail.